norddrop/src/ffi/mod.rs

```diff
@@ -119,6 +119,7 @@ pub unsafe extern "C" fn norddrop_new_transfer(
 #[no_mangle]
 pub unsafe extern "C" fn norddrop_destroy(dev: *mut norddrop) {
     if !dev.is_null() {
+        #[allow(clippy::let_underscore_must_use)]
         let _ = Box::from_raw(dev);
     }
```

Comment:
we could do `let inst = Box::from_raw(dev);` this way the lint comment would not be needed, however it's one occasion, it doesn't really matter

Reply:
then we'd have to use `allow unused variables` instead of this to make the lint stage pass :D